src/containerapp/azext_containerapp/commands.py

```diff
@@ -45,6 +45,7 @@ def load_command_table(self, args):
         g.custom_show_command('show', 'show_containerappsjob')
         g.custom_command('list', 'list_containerappsjob')
         g.custom_command('create', 'create_containerappsjob', supports_no_wait=True, exception_handler=ex_handler_factory(), transform=transform_sensitive_values)
+        g.custom_command('update', 'update_containerappsjob', supports_no_wait=True, exception_handler=ex_handler_factory(), transform=transform_sensitive_values)
```

Comment:
It appears to be a customer-facing change. If so, please add changelog to the HISTORY.rst file.

Reply:
Done. This command already support in Azure-cli, we support it in extension to do the refactor first.